doc/api/net.md

```diff
@@ -1069,6 +1069,8 @@ For TCP connections, available `options` are:
 * `noDelay` {boolean} If set to `true`, it disables the use of Nagle's algorithm
   immediately after the socket is established. **Default:** `false`.
 * `port` {number} Required. Port the socket should connect to.
+* `blockList` {net.BlockList} `blockList` can be used for disabling outbound
```

Comment:
Likely easiest if this were all lower-case
```suggestion
* `blocklist` {net.BlockList} `blocklist` can be used for disabling outbound
```

Reply:
done.

Reply:
This is inconsistent with the name of the class

Reply:
Do you mean blocklist is different from BlockList？

Reply:
Yes. If the class was called `Blocklist` I would understand the suggestion but here it find it illogical

Reply:
I think `blockList` is better, I have updated `blocklist` to `blockList`. Thanks ! cc @jasnell .